src/font/mod.rs

```diff
@@ -4,12 +4,17 @@ use core::ops::Deref;
 
 pub(crate) mod fallback;
 
-pub use self::matches::*;
-mod matches;
-
 pub use self::system::*;
 mod system;
 
+#[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+/// Identifies a [`Font`] in a [`FontSystem`]
+pub struct FontKey {
+    pub id: fontdb::ID,
+    #[cfg(feature = "swash")]
+    pub swash: (u32, swash::CacheKey),
```

Comment:
This potentially should be repr(transparent) if swash isn't active (I forgot if you are allowed to use transparent with multiple fields)

Reply:
Done.